test/integration/contract-aci.js

```diff
@@ -145,9 +145,9 @@ describe('Contract instance', function () {
 
   before(async function () {
     aeSdk = await getSdk()
-    testContractAci = await aeSdk.compilerApi
+    testContractAci = await aeSdk.contractCompiler.compilerApi
```

Comment:
the approach looks good, but I would combine `ContractCompilerHttp` and `CompilerApi`, to have just:
```suggestion
    testContractAci = await aeSdk.compilerApi
```